common/components/WalletDecrypt/components/Mnemonic.tsx

```diff
@@ -1,11 +1,13 @@
 import { mnemonicToSeed, validateMnemonic } from 'bip39';
-import DPATHS from 'config/dpaths';
 import React, { Component } from 'react';
 import translate, { translateRaw } from 'translations';
 import DeterministicWalletsModal from './DeterministicWalletsModal';
 import { formatMnemonic } from 'utils/formatters';
+import { ETH_DEFAULT } from 'config/dpaths';
+import { getPaths } from 'selectors/config';
+import { InsecureWalletName } from 'config';
 
-const DEFAULT_PATH = DPATHS.MNEMONIC[0].value;
+const DEFAULT_PATH = ETH_DEFAULT.value;
```

Comment:
Need to use selector here as well to get default mnemonic path instead of assuming ether (for other networks)